client/src/components/header.js

```diff
@@ -0,0 +1,70 @@
+import React, {useState} from 'react';
+import HomeOutlinedIcon from '@material-ui/icons/HomeOutlined';
+import AccountTreeOutlinedIcon from '@material-ui/icons/AccountTreeOutlined';
+import AccountCircleOutlinedIcon from '@material-ui/icons/AccountCircleOutlined';
+
+export default function Header() {
+
+    // Change this to toggle between logged in and logged out
+    const [loggedIn, setLoggedIn] = useState(true)
+
+    //Roles are Supplier, Designer, Contractor
+    const [role, setRole] = useState("Supplier")
+    return (
+        <div className='header'>
+            {loggedIn ? (
```

Comment:
I think this should be `!loggedIn` ? Just want to confirm

Reply:
Oooh very true, good catch thank you!